app/controllers/admin/rebate_forms_controller.rb

```diff
@@ -1,9 +1,20 @@
 # frozen_string_literal: true
 
+require 'rqrcode'
+require 'base64'
+require 'jwt'
+
 class Admin::RebateFormsController < Admin::BaseController
   before_action :set_rebate_form, only: %i[show update destroy edit]
   respond_to :html, :pdf, :csv, :json
 
+  def generateqr
```

Comment:
Is there a test for this?

Reply:
should this be `generate_qr` ? :elephant: